server/routes/contacts/manage/prisoner-search/prisonerSearchResultsController.test.ts

```diff
@@ -0,0 +1,92 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { SessionData } from 'express-session'
+import { v4 as uuidv4 } from 'uuid'
+import { appWithAllRoutes, user } from '../../../testutils/appSetup'
+import AuditService, { Page } from '../../../../services/auditService'
+import PrisonerSearchService from '../../../../services/prisonerSearchService'
+import { PaginationRequest, Prisoner } from '../../../../data/prisonerOffenderSearchTypes'
+import logger from '../../../../../logger'
+
+jest.mock('../../../../services/auditService')
+jest.mock('../../../../services/prisonerSearchService')
+
+const auditService = new AuditService(null) as jest.Mocked<AuditService>
+const prisonerSearchService = new PrisonerSearchService(null) as jest.Mocked<PrisonerSearchService>
+
+let app: Express
+let session: Partial<SessionData>
+const journeyId: string = uuidv4()
+
+beforeEach(() => {
+  app = appWithAllRoutes({
+    services: {
+      auditService,
+      prisonerSearchService,
+    },
+    userSupplier: () => user,
+    sessionReceiver: (receivedSession: Partial<SessionData>) => {
+      session = receivedSession
+      session.prisonId = 'HEI'
+      session.manageContactsJourneys = {}
+      session.manageContactsJourneys[journeyId] = {
+        id: journeyId,
+        lastTouched: new Date(),
+        search: {
+          searchTerm: 'test',
+        },
+      }
+    },
+  })
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET /contacts/manage/prisoner-search-results', () => {
+  it('should render the prisoner search results page', async () => {
+    auditService.logPageView.mockResolvedValue(null)
+    prisonerSearchService.searchInCaseload.mockResolvedValue({
+      totalPages: 1,
+      totalElements: 1,
+      first: true,
+      last: true,
+      size: 1,
+      content: [
+        {
+          lastName: 'test',
+          firstName: 'test',
+          prisonerNumber: 'test',
+          dateOfBirth: '2000-01-01',
+        } as Prisoner,
+      ],
+    })
+
+    const response = await request(app).get(`/contacts/manage/prisoner-search-results/${journeyId}`)
+
+    logger.info(`Response = ${JSON.stringify(response)}`)
```

Comment:
is this intentional? we could reduce the noise on the test console output by removing `logger.info(`Response = ${JSON.stringify(response)}`)`

Reply:
Since removed in next PR.